packages/react-components/react-search/docs/MIGRATION.md

```diff
@@ -0,0 +1,94 @@
+| label position               | `labelPosition?: 'above' \| 'inline' \| 'inside'`           | handled by Field                        |
+| description                  | use FormField                                               | handled by Field                        |
+| error message                | use FormField                                               | handled by Field                        |
+| icon at start of field       | `icon?: ShorthandValue<BoxProps>` + `iconPosition: 'start'` | `contentBefore` slot                    |
+| icon at end of field         | `icon?: ShorthandValue<BoxProps>` + `iconPosition: 'end'`   | `contentAfter` and `dismiss` slots      |
+| value                        | `value?: string \| number`                                  | `value?: string`                        |
+| defaultValue                 | `defaultValue?: string \| string[]`                         | `defaultValue?: string`                 |
+| onChange                     | `(ev, data: { ...props, value }) => void`                   | `(ev, data: { value: string }) => void` |
+| onClear                      | `(ev?: any) => void`                                        | `(ev?: any) => void` on `dismiss` slot  |
+| onSearch                     | `(newValue: any) => void`                                   | not supported                           |
+| container className          |                                                             | top-level `className`                   |
+| input className              | `input.className?: `                                        |                                         |
+| aria label                   | `'aria-label'?: string`                                     | `'aria-label'?: string`                 |
+| clearable                    | `clearable?: boolean`                                       | `dismiss` slot                          |
+| style overrides              | `styles?: ComponentSlotStyles<...>`                         | className, slot classNames              |
+| accessibility                | individual props                                            | individual props                        |
+
+```ts
+// packages/fluentui/react-northstar/src/utils/htmlPropsUtils.tsx
+type SupportedIntrinsicInputProps = {
+  [K in HtmlInputProps]?: K extends keyof JSX.IntrinsicElements['input'] ? JSX.IntrinsicElements['input'][K] : any;
+};
+// HtmlInputProps: a subset of React and HTML native props
+
+// packages/fluentui/react-northstar/src/components/Box/Box.tsx
+interface BoxProps extends UIComponentProps<BoxProps>, ContentComponentProps, ChildrenComponentProps {
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility<never>;
+}
+```
+
+## Migrations from v0
+
+| Prop/concept                 | v8                                                    | v9                                      |
```

Comment:
It looks like a couple v8 props might be missing: `onEscape` and `disableAnimation`